### DISCUSSION TITLE
🔗 Add Telegram to Floating Buttons (Duplicate of #28090)

### DISCUSSION THREAD

**Discussion by parfilov** [elementor/elementor#28153] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Adding a link to a Telegram profile

### Describe the solution you'd like.

It would be nice to have Telegram on the list of platforms

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by sepidev** [comment]
It is really needed

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #28090

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
